packages/react-native/Libraries/NewAppScreen/components/Header.js

```diff
@@ -42,7 +42,7 @@ const Header = (): Node => {
         ]}>
         Welcome to
         {'\n'}
-        React Native
+        React Native visionOS
       </Text>
     </ImageBackground>
```

Comment:
we wanted to change the image background to glass, right?

Reply:
Yeah, it's this one 😅 I will change it in this PR